Initializer/Initialize.cs

```diff
@@ -243,32 +315,39 @@ private void _writeCategoriesToDataBase()
                 catch (InvalidOperationException)
                 {
                     _isData = false;
-                    Console.WriteLine("Data isn't exist");
+                    Console.WriteLine("Data doesn't exist");
                 }
                 catch (Exception e)
                 {
                     Console.WriteLine(e);
                 }
             }
 
-            //Save data to dataBase
-            _manageAccessToEntity.CategoryTranslations.CreateRange(allCategoryTranslations);
+            //Take only new data and save data to dataBase
+            List<string> existingCategoryTranslations = _manageAccessToEntity.CategoryTranslations.
+                GetAll().Select(c => c.CategoryTranslationName).ToList();
+            _manageAccessToEntity.CategoryTranslations.CreateRange(allCategoryTranslations.Where(c => !existingCategoryTranslations
+                .Contains(c.CategoryTranslationName)).ToList());
             Console.WriteLine("{0}/8. Category translations have added", ++_countStage);
         }
 
         private void _writeSubCategoriesToDataBase()
         {
+            //Get existing data
+            List<string> existingSubCategories = _manageAccessToEntity.SubCategories.
```

Comment:
Please give more meaningful names to variables - e.g. "existingSubCategoryNames" instead of "existingSubCategories "